bin/check-load.rb

```diff
@@ -28,11 +28,15 @@
 class LoadAverage
   def initialize(per_core = true)
     @cores = per_core ? cpu_count : 1
-    @avg = File.read('/proc/loadavg').split.take(3).map { |a| (a.to_f / @cores).round(2) } rescue nil # rubocop:disable RescueModifier
+    @avg = `uptime`.delete(',').split(' ')[-3..-1].map { |a| (a.to_f / @cores).round(2) } rescue nil # rubocop:disable RescueModifier
```

Comment:
I copied that pattern from https://github.com/sensu-plugins/sensu-plugins-load-checks/blob/master/bin/metrics-load.rb#L61, but I'll change it to do a fallback, since BSD doesn't have the /proc/ interface by default.